src/types/voice-channel.js

```diff
@@ -7,11 +7,11 @@ class VoiceChannelArgumentType extends ArgumentType {
 		super(client, 'voice-channel');
 	}
 
-	validate(val, msg, arg) {
+	async validate(val, msg, arg) {
```

Comment:
```suggestion
	validate(val, msg, arg) {
```
as per last comment this doesn't need to be async